app/assets/javascripts/utils/ChatContainer.js

```diff
@@ -9,7 +9,7 @@ const nullEventHandler = {
     onConfirmEndChat: function () {},
     onSoundToggle: function () {},
     onStartTyping: function () {},
-    onStopTyping: function () {}
+    onStopTyping: function () {},
```

Comment:
Does this comma keep turning up because automatic formatting is doing it? Seems a little odd.

Reply:
yes it did, i'll remove